apisix/debug.lua

```diff
@@ -123,15 +124,35 @@ local function apple_new_fun(module, fun_name, file_path, hook_conf)
 
     function mt.__call(self, ...)
         local arg = {...}
+        local http_filter = debug_yaml.http_filter
+        local api_ctx = ngx.ctx.api_ctx
         if hook_conf.is_print_input_args then
-            log[log_level]("call require(\"", file_path, "\").", fun_name,
-                           "() args:", json.delay_encode(arg, true))
+            if not (http_filter and http_filter.enable)
+                    or (api_ctx and api_ctx.enable_dynamic_debug) then
```

Comment:
Better to refactor the complex expression to a boolean variable

Reply:
done